### DISCUSSION TITLE
How to make a userstyles addon

### DISCUSSION THREAD

**Discussion by cioudo** [ScratchAddons/ScratchAddons#7353] [Help] [answered]
Hi! I've been trying to make a userstyles addon these days, but it won't work. I've made a manifest and a styles.css, wich should be correct. Can someone help me?

Thanks,
@cioudo (@wioudo on Scratch becouse banned)

**Comment by jazzathedev** [comment]
Please send the code you have in some way shape or form, either send to here, to the dev server or DM to me

**Reply by cioudo** [reply]
manifest.json:
```
{
    "manifest_version": 3,
    "name": "Apply other fonts to scratch.mit.edu.",
    "version": "1.0",
    "description": "pply other fonts to scratch.mit.edu.",
    "permissions": ["activeTab"],
    "content_scripts": [
      {
        "matches": ["https://scratch.mit.edu/*"],
        "css": ["styles.css"]
      }
    ]
  }
  ```
styles.css:
```
[class*="text_"] {
    font-family: monospace;
  }
  ```

**Reply by jazzathedev** [reply, marked as answer]
Please refer to https://scratchaddons.com/docs/develop/getting-started/creating-an-addon/ for a general tutorial, https://scratchaddons.com/docs/reference/addon-manifest/ for a general idea of what the addon.json should look like and how to link to a userstyle file (it seems you have made an extension manifest)

**Reply by WorldLanguages** [reply]
@cioudo Your chrome extension seems fine, but we can't help with that here.
